src/model.py

```diff
@@ -345,8 +345,20 @@ def __init__(
         # init after kwargs stored
         self.unpack_lrs()
 
+        # for i in range(5):  # dirty fix to avoid intermittend HTTPError for hugging face
+        #     try:
+        #         self.bert = BertModel.from_pretrained(bert_path).to(torch.device(self.device))
+        #     except Exception as e:
+        #         if "HTTPError" in e.msg() and i<4:
+        #             continue
+        #         raise e 
+        #     if self.find("bert") is not None:
+        #         break
```

Comment:
Could be an idea to build a safer loader form hugging face. Does not fail ofte, but annoying when it does.